test/index.test.js

```diff
@@ -268,10 +268,7 @@ describe('Plugin', function () {
           path: 'tmp',
           integrity: true
         }),
-        new SriPlugin({
-          hashFuncNames: ['sha384'],
-          enabled: true
-        })
+        new SubresourceIntegrityPlugin()
```

Comment:
https://github.com/waysact/webpack-subresource-integrity/releases/tag/v5.0.0 brings new defaults.

Configuration is not needed anymore, they even recommend using it like that https://github.com/waysact/webpack-subresource-integrity/blob/main/MIGRATE-v1-to-v5.md#new-usage-suggested-defaults-recommended